src/aura/STG_CMP_System/STG_CMP_SystemHelper.js

```diff
@@ -30,6 +30,51 @@
 			$A.util.removeClass(errNoticeUserId, "slds-show");
 			$A.util.addClass(errNoticeUserId, "slds-hide");
 		}
-	}
+	}, 
+    
+    handleRefreshHouseholdAccount: function(component, event, helper) {
+    	var action = component.get("c.executeRefreshHouseholdAccountBatch"); 
+        action.setCallback(this, function(response) {
+                            var state = response.getState(); 
+        if (state === "SUCCESS") {            
+            var toast = component.find("hhSuccessToast"); 
+        	$A.util.removeClass(toast,'slds-hide');
+            $A.util.addClass(toast, 'slds-show'); 
+        } else if (state ==="INCOMPLETE") {
+            console.log('InsideHelperforHandleRefresh Incomplete'); 
+        } else if (state ==="ERROR") {
+            console.log('InsideHelperforHandleRefresh ERROR'); 
+        }
+        }); 
+        
+        $A.enqueueAction(action); 
+    },
+    
+    handleRefreshAdminAccount: function(component, event, helper) {
+    	var action = component.get("c.executeRefreshAdminAccountBatch"); 
+        action.setCallback(this, function(response) {
+                            var state = response.getState(); 
+        if (state === "SUCCESS") {            
+            var toast = component.find("adminSuccessToast"); 
+        	$A.util.removeClass(toast,'slds-hide');
+            $A.util.addClass(toast, 'slds-show'); 
+        } else if (state ==="INCOMPLETE") {
+            console.log('InsideHelperforHandleRefresh Incomplete'); 
```

Comment:
if we don't actually act on/display errors, etc lets not even check for them. I don't believe the packaging folks want us to have console.log stuff in our components.

Reply:
or we can check but we can't console.log them...